### PR TITLE
Set continuation indent to 4 in IntelliJ explicitly

### PR DESCRIPTION
**Pull request by lptr**
For some reason IntelliJ keeps occasionally switching between 4 and 8 otherwise.

### CODE REVIEW

File: .idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
-    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```
Comment (wolfs) [inline .idea/codeStyles/codeStyleConfig.xml:4]: ❓ I am still unsure why this file changes sometimes and sometimes doesn't. Should we exclude the changes from this PR?

Comment (wolfs) [reply .idea/codeStyles/codeStyleConfig.xml:4]: Maybe `Project.xml` is the default for project code style? Let's remove the line again from this PR and only change the other file.

**Review by jvandort — APPROVED** [review]

File: .idea/codeStyles/codeStyleConfig.xml (hunk 1 above)
Comment (wolfs) [reply .idea/codeStyles/codeStyleConfig.xml:4]: Alright, fine with me as well. FWIW, the GE project has `<option name="PREFERRED_PROJECT_CODE_STYLE" value="Project" />` in there. Is that maybe what we should do as well?

**Comment by lptr** [comment]
@bot-gradle test and merge please.

File: .idea/codeStyles/codeStyleConfig.xml (hunk 1 above)
Comment (lptr) [reply .idea/codeStyles/codeStyleConfig.xml:4]: I don't feel strongly. This seems to work well. If we bump into more problems, let's re-assess.

**Pull request merged**